omim2obo/main.py

```diff
@@ -217,7 +217,11 @@ def omim2obo(use_cache: bool = False):
         graph.add((OMIM[mim_number], SKOS.exactMatch, NCBIGENE[entrez_id]))
     for mim_number, entrez_id in pheno_map.items():
         # RO['0002200'] = 'has phenotype'
-        graph.add((NCBIGENE[entrez_id], RO['0002200'], OMIM[mim_number]))
+        b = BNode()
+        graph.add((b, RDF['type'], OWL['Restriction']))
+        graph.add((b, OWL['onProperty'], RO['0002200']))
+        graph.add((b, OWL['someValuesFrom'], OMIM[mim_number]))
+        graph.add((NCBIGENE[entrez_id], RDFS['subClassOf'], b))
```

Comment:
This makes sure the relation is rendered correctly in OWL as an existential restriction.

Reply:
Exactly the opposite: an existential restriction (which is the only way to model class-level relations in OWL), means that "all instances of gene `NCBIGENE[entrez_id]` are `RO['0002200']`-related to at least 1 instance of `OMIM[mim_number]`. So all instances of NCBIGene:123 are related to at least one instance of OMIM:123.

Reply:
ahhh, ok thanks